src/core/server/models/story/story.ts

```diff
@@ -765,6 +765,26 @@ function getMarkStoryForArchivingSetParam(now: Date) {
   };
 }
 
+export async function forceMarkStoryForArchiving(
```

Comment:
I think it would be helpful to add comments and/or otherwise refactor/rename for clarity regarding the difference between `forceMarkStoryForArchiving` and `markStoryForArchiving`.